scripts/ci/images/el7/Dockerfile

```diff
@@ -29,6 +29,17 @@ ENV PATH /opt/sz/2.1.8.3/bin:${PATH}
 ENV LD_LIBRARY_PATH /opt/sz/2.1.8.3/lib64:${LD_LIBRARY_PATH}
 ENV CMAKE_PREFIX_PATH /opt/sz/2.1.8.3:${CMAKE_PREFIX_PATH}
 
+#Install DataSpaces
+WORKDIR /opt/dspaces
+RUN curl -L https://dataspaces.rdi2.rutgers.edu/downloads/dataspaces-1.8.0.tar.gz | tar -xvz && \
+    mkdir install
+    cd 1.8.0
+    ./configure CC=mpicc CXX=mpic++ FC=mpifort --prefix=/opt/dspaces/1.8.0
+    make -j
```

Comment:
The `el7` build is without MPI so it needs to go in a different image.  Also using `make -j` instead of `make -jN` means `j = infinite`, not great.  See the above SZ build for using the correct `j` value.